cmd/root.go

```diff
@@ -90,7 +86,7 @@ func PreRun(cmd *cobra.Command, args []string) {
 	lifecycleHooks, _ = f.GetBool("enable-lifecycle-hooks")
 
 	// configure environment vars for client
-	err := flags.EnvConfig(cmd, DockerAPIMinVersion)
+	err := flags.EnvConfig(cmd)
```

Comment:
Some test confirming that this works would be awesome.

Reply:
Tests added.